recipes/uppasd/meta.yaml

```
@@ -0,0 +1,104 @@
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja
+  host:
+    - fftw  # [not win]
+    - libblas  # [not win]
+    - liblapack  # [not win]
+    - mkl-devel >=2025.2.0  # [win]
+    - llvm-openmp {{ fortran_compiler_version }}.*  # [win]
+    - llvm-openmp-fortran {{ fortran_compiler_version }}.*  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  run:
+    - libblas * *mkl  # [win]
+    - mkl >=2025.2.0  # [win]
+
+test:
+  source_files:
+    - examples/SimpleSystems/bccFe
+  commands:
+    - cd examples/SimpleSystems/bccFe && uppasd  # [not win and build_platform == target_platform]
+    - cd examples/SimpleSystems/bccFe && uppasd.exe  # [win]
+
+# Uncommenting the outputs section would be preferable but it breaks the build process with:
+# ...
+# Packaging uppasd
+# Packaging uppasd-v6.0.2-ha770c72_0
+# number of files: 1
+# Warning: rpath /home/conda/staged-recipes/build_artifacts/uppasd-split_1764156508131/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/uppasd-split_1764156508131/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol (removing it)
+#   ERROR (uppasd,bin/uppasd): $RPATH/libgfortran.so.5 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): $RPATH/libblas.so.3 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): $RPATH/liblapack.so.3 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): /lib64/libpthread.so.0 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): $RPATH/libfftw3.so.3 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): /lib64/libgomp.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): /lib64/libm.so.6 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): /lib64/libgcc_s.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+#   ERROR (uppasd,bin/uppasd): /lib64/libc.so.6 not found in packages, sysroot(s) nor the missing_dso_whitelist.
+# .. is this binary repackaging?
+# 
+# outputs:
+#   # Only distribute the binary but not the static library. It does not have a publicly
+#   # documented API.
+#   - name: uppasd
+#     files:
+#       include:
+#         - bin/uppasd  # [not win]
+#         - Library/bin/uppasd.exe  # [win]
+#       exclude:
+#         - lib/libasdlib.a  # [not win]
+#         - Library/lib/libasdlib.a  # [win]
```

To get the outputs to work you need at least a `build:` section, to inherit the right run-exports that take care of the sysroot, etc.

```suggestion
outputs:
  - name: uppasd
    requirements:
      build:
        - {{ compiler('fortran') }}
        - {{ stdlib('c') }}
    files:
      include:
        - bin/uppasd  # [not win]
        - Library/bin/uppasd.exe  # [win]
      exclude:
        - lib/libasdlib.a  # [not win]
        - Library/lib/libasdlib.a  # [win]
```

However, your comment "Only distribute the binary but not the static library" is not clear to me; is there really nothing else being shipped here than the binary? If so you'd need another output of course.

Given that everything is being built in one go, you could also rename the build scripts to something other than `build.sh` / `bld.bat` and then move all the requirements to the outputs, and do
```
outputs:
  - name: uppasd
    script: build-sd.sh   # [unix]
    script: build-sd.bat  # [win]
    requirements:
      build:
        - ...
```

---

sounds good